Demos/Basic-Swift/Basic-Swift/SampleCollectionViewController.swift

```diff
@@ -52,64 +38,60 @@ class SampleCollectionViewController: UICollectionViewController {
     let validateButtonItem = UIBarButtonItem(title: "Validate",
       style: .Done,
       target: self,
-      action: NSSelectorFromString("validateButtonAction"))
+      action: "validateButtonAction")
 
     let updateButtonItem = UIBarButtonItem(title: "Update",
       style: .Done,
       target: self,
-      action: NSSelectorFromString("updateButtonAction"))
+      action: "updateButtonAction")
 
-    let flexibleBarButtonItem = UIBarButtonItem(barButtonSystemItem: .FlexibleSpace,
-      target: nil,
-      action: nil)
+    let flexibleBarButtonItem = UIBarButtonItem(barButtonSystemItem: .FlexibleSpace)
 
-    let readOnlyView = UIView(frame: CGRectMake(0.0, 0.0, 150.0, 40.0))
-    let readOnlyLabel = UILabel(frame: CGRectMake(0.0, 0.0, 90.0, 40.0))
+    let readOnlyView = UIView(frame: CGRectMake(0, 0, 150, 40))
+    let readOnlyLabel = UILabel(frame: CGRectMake(0, 0, 90, 40))
     readOnlyLabel.text = "Read-Only"
     readOnlyLabel.textColor = UIColor(fromHex: "5182AF")
-    readOnlyLabel.font = UIFont.boldSystemFontOfSize(17.0)
+    readOnlyLabel.font = UIFont.boldSystemFontOfSize(17)
```

Comment:
float, because of type inference, you don't need to specify it, compiler understands it
CGRect init(x: CGFloat, y: CGFloat, width: CGFloat, height: CGFloat)